src/Sylius/Component/Core/Model/AvatarAwareInterface.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Component\Core\Model;
+
+interface AvatarAwareInterface
```

Comment:
I think we don't need this interface - let's just use `AdminUserInterface` instead.